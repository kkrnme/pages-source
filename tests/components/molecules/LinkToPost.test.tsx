import React from "react"
import renderer from "react-test-renderer"
import {
  LinkToPost,
  LinkToPostProps,
} from "../../../src/components/molecules/LinkToPost"
import { css } from "@emotion/core"

const tests = [
  {
    excerpt: "ああああああああああああああああああ",
    title: "今日の天気",
    path: "/",
    css: css``,
  },
]

describe(`LinkToPost`, () => {
  for (const i of tests) {
    it(`renders correctly ${i}`, () => {
      const tree = renderer.create(<LinkToPost post={i}></LinkToPost>).toJSON()

      expect(tree).toMatchSnapshot()
    })
  }
})
