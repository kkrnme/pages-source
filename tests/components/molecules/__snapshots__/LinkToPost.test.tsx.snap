// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkToPost renders correctly [object Object] 1`] = `
<a
  aria-current="page"
  className="css-1yrbkum"
  href="/"
  onClick={[Function]}
  onMouseEnter={[Function]}
  style={Object {}}
>
  <div>
    <h2>
      <span>
        今日の天気
      </span>
    </h2>
  </div>
  <p>
    ああああああああああああああああああ
  </p>
</a>
`;
